src/settings/portalSettings.ts

```diff
@@ -8,40 +8,37 @@ import { SPACE_TAXON } from "settings/taxonomy";
 
 import { RoomVisibility } from "types/venues";
 
-type VisibilityOption = {
+export type VisibilityLabelOption = {
   label: string;
   value: RoomVisibility;
   subtitle?: { text: string; icon: IconDefinition }[];
 };
 
-export type LabelVisibilityOptions = {
-  noLabel: VisibilityOption;
-  hoverTitle: VisibilityOption;
-  countOnly: VisibilityOption;
-  titleAndCount: VisibilityOption;
-};
-
-export const LABEL_VISIBILITY_OPTIONS: LabelVisibilityOptions = {
-  noLabel: {
+export const VISIBILITY_LABEL_OPTIONS: Record<
+  // @debt: please add a clarification (or link to such) as to why unclickable is excluded
```

Comment:
not sure about this one?

Reply:
Well, I'm sure about that one, it's a good @debt comment, maybe the answer will come some day :D